package.json

```diff
@@ -50,7 +50,9 @@
   "dependencies": {
     "favicons": "^4.8.3",
     "loader-utils": "^0.2.14",
-    "lodash": "^4.11.1",
-    "webpack": "^1.13.0"
+    "lodash": "^4.11.1"
+  },
+  "peerDependencies": {
+    "webpack": "1 || ^2.1.0-beta"
```

Comment:
Can you please updating to the release webpack v2 here?
@LoicMahieu  Can you please take a look at it? Thank you.